README.md

```diff
@@ -194,18 +251,16 @@ Loading...
 </style>
 ```
 
-In an app that contains matching views, the developer does not have to worry about specifics of the navigation API, or how content is streamed.
-All they have to do is match their views with URL routes, and then stream the appropriate new content wrapped in `<template for=... mode...>` snippets.
-#### Details of `<view>`
+In an app that contains matching views, the developer does not have to worry about specifics of the navigation API, or the mechanics of streaming/piping.
+All they have to do is list their routes, declare which parts of their document match their URL routes, and then stream the appropriate new content wrapped in `<template for=...>` snippets.
+
+#### Details of declarative interception
 
-1. Multiple views can be present at the same time, and updated together using the same response. The response can be *identical* to a full document navigation response, as long as the view content is wrapped in `<template for>`. 
-1. A `view` would also have lifecycle events to enable developers to fine-tune its behavior using JS.
+1. When a route has a `mode: 'same-document'` clause, navigations to matching destinations would be intercepted. The navigation request would still take place, but only `<template for>` elements from the response would be spliced-streamed into the document.
```

Comment:
Are additional HTTP headers needed for this request, so that the server can optionally respond with just the `<template>` elements for this kind of request?